tasks/tasks/settings.py

```diff
@@ -6,6 +6,8 @@ class Settings(BaseSettings):
     ELASTICSEARCH_URL: str = "http://localhost:9200"
 
     DATASET_STORAGE_BASE_URL: str
+    DOCUMENT_STORAGE_BASE_URL: str
```

Comment:
This is now needed- same value as on the API side (will be used from common stack .envfile).

Reply:
We may be able to improve on this for the external OCR_URL process in the future, but for now this will do.